src/Cli/Program.cs

```diff
@@ -4,6 +4,7 @@
 using CommandLine;
 using Microsoft.Extensions.Logging;
 using static Cli.Utils;
+using static Azure.DataApiBuilder.Service.DABUtils;
```

Comment:
May be keep the class name in the service `Utils.cs` but locally within this file call it as `DABUtils` to avoid collision with the Cli.Utils?

Reply:
Any issues with DABUtils as the file name?

Reply:
Theres already DataApiBuilder in the namespace name, - DABUtils seems redundant usage of DAB

Reply:
Hmm sounds ok. Will change

Reply:
Done